archived/resources/OperationDefinition-deqm.evaluate-measure.xml

```diff
@@ -44,6 +44,13 @@
         <type value="date"/>
     </parameter>
     <parameter>
+      <name value="program"/>
+      <use value="in"/>
+      <min value="0"/>
+      <max value="1"/>
+      <documentation value="The quality program being reported via canonical reference to ([Library{CQFMQualityProgram}](http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/library-cqfm)) indicating which quality program is being reported."/>
```

Comment:
https://jira.hl7.org/browse/FHIR-45309 renames CQFMQualityProgram to CQFMContentRelease, not yet applied